### DISCUSSION TITLE
Add BMW logo or car image for HA Map

### DISCUSSION THREAD

**Discussion by Ogurtsov-D** [bimmerconnected/bimmer_connected#322] [Feature request] [answered]
Hi, 

In my HA map the car position is marked with strange "Xx".  
![Без имени-1](https://user-images.githubusercontent.com/78870757/137809046-d5414edf-075c-4eb4-ba28-7ddf2c7a508b.jpg)
It would be nice replace it with BMW logo or just allow to upolad your own picture, similar to device tracker of a person:
![Без имени-2](https://user-images.githubusercontent.com/78870757/137809914-a22cc45a-325a-4b50-8232-9f6f8417d339.jpg)

**Comment by rikroe** [comment, marked as answer]
I think adding an Icon to a `device_tracker` is not possible in HA. You could either add the BMW to your own person or create a new person called "Car" (or whatever) with BMW logo as image.

If this doesn't suit your needs, please also ask in the HA forums/HA issue tracker.
